pkg/k8s/watchers/pod.go

```diff
@@ -765,10 +765,10 @@ func (k *K8sWatcher) updatePodHostData(oldPod, newPod *slim_corev1.Pod, oldPodIP
 			if !errors.Is(err, &ipcache.ErrOverwrite{
 				ExistingSrc: source.KVStore,
 				NewSrc:      source.Kubernetes,
-			}) || !errors.Is(err, &ipcache.ErrOverwrite{
+			}) && !errors.Is(err, &ipcache.ErrOverwrite{
```

Comment:
Could you add code commets saying why it's OK to ignore these errors?

Reply:
@brb did you check already existing lines 758-764? :wink: